backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleRunnerClient.java

```diff
@@ -247,6 +238,11 @@ public void runPlaybook(List<String> command, int timeout, String uuid) throws E
     protected String formatCommandVariables(Map<String, Object> variables, String playAction) {
         String result;
         try {
+            ObjectMapper mapper = JsonMapper
+                .builder()
+                .findAndAddModules()
+                .build()
+                .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
```

Comment:
What's so harmful on having JsonMapper shared between threads? Unless I'm missing something we use that approach a lot in engine, because JsonMapper initialization takes some time ...

Reply:
Right, I thought sharing the mapper between threads might be an issue, but found in the docs: `Mapper instances are fully thread-safe provided that ALL configuration of the instance occurs before ANY read or write calls`. Thanks!